run_release.py

```diff
@@ -533,7 +533,10 @@ def check_pyspecific(db: ReleaseShelf) -> None:
         f"SOURCE_URI = 'https://github.com/python/cpython/tree/{expected_branch}/%s'"
     )
     if expected != line.strip():
-        raise ReleaseException("SOURCE_URI is incorrect")
+        raise ReleaseException(
+            f"SOURCE_URI is incorrect, expected: {expected}, got: {line.strip()} "
+            "(it needs changing before beta 1)"
```

Comment:
I usually line up things so it's clear what parts don't match.  In this case it should be pretty easy to spot, but given that the error message is quite long and might wrap around, you might still prefer doing something like:
```suggestion
            f"SOURCE_URI is incorrect (it needs changing before beta 1):\n"
            f"expected: {expected}\n"
            f"got     : {line.strip()}"
```